src/main/java/org/dependencytrack/resources/v1/VulnerabilityResource.java

```diff
@@ -173,27 +178,74 @@ public Response getVulnerabilityByVulnId(@PathParam("source") String source,
     }
 
     @GET
-    @Path("/source/{source}/vuln/{vuln}/projects")
+    @Path("/source/{source}/vuln/{vuln}/includeSuppressed/{includeSuppressed}/analysis")
```

Comment:
I have resolved this suggestion by following way:

Leave the "/source/{source}/vuln/{vuln}/projects" as it is (Inorder to avoid the breaking existing API).
Introduce new API '/source/{source}/vuln/{vuln}/dependencyAnalysis' with optional parameter to achieve above use case.

**Reason to introduce new API.**
   "/source/{source}/vuln/{vuln}/projects" API returns Project model. Since Project model does not have analysis metadata, I have introduced above mentioned new API which returns DependencyAnalysisResponse.

If you have any concerns about this, please let me know. :)